Formula/erlang.rb

```diff
@@ -2,8 +2,8 @@ class Erlang < Formula
   desc "Programming language for highly scalable real-time systems"
   homepage "https://www.erlang.org/"
   # Download tarball from GitHub; it is served faster than the official tarball.
```

Comment:
Not sure how true this still is. Could potentially be slower because we have to download and install autoconf etc when not using the release tarball.

Reply:
Yea, that comment is about a decade old. Probably not true anymore.